source/manual/setting-up-new-rails-app.html.md

```diff
@@ -8,140 +8,161 @@ last_reviewed_on: 2018-03-22
+production:
+  <<: *default
+  database: myapp_production
+  url: <%= ENV["DATABASE_URL"]%>
+```
 
-Be sure to use the same port number which you assigned in the
-`Procfile`.
+In config, replace `credentials.yml.env` and `master.key` with `secrets.yml`.
 
-## Setting up DNS entries
+```
+development:
+  secret_key_base: secret
 
-If the app requires the creation of new external (publicly accessible)
-subdomains, you can do this via [the govuk-dns-config
-repo](https://github.com/alphagov/govuk-dns-config). If you need assistance
-with pointing the records at the correct boxes, talk to Reliability
-Engineering.
+test:
+  secret_key_base: secret
 
-## Adding deployment scripts
+production:
+  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
+```
 
-After the app has been configured and defined in Puppet, the deployment
-scripts are the next step to getting the app running on the boxes.
-Deployment scripts are bespoke to each application, but they live
-together in the [govuk-app-deployment
-repository](https://github.com/alphagov/govuk-app-deployment).
+In config, replace the content of `routes.rb` with the following healthcheck.
 
-All Rails apps use [Capistrano](http://www.capistranorb.com/) for deployment, and
-common deployment steps have been extracted into the `recipes` directory.
+```
+Rails.application.routes.draw do
+  get "/healthcheck", to: proc { [200, {}, ["OK"]] }
+end
+```
 
-It's recommended to copy the deploy scripts from an existing app as a
-starting point. The deployment scripts for the `content-store` are
-fairly generic and have little custom behaviour, which may be useful.
+Now is a good time to run `bin/setup`. Lastly, create `lib/tasks/lint.rake` with this.
 
-> **warning**
->
-> If the names of the app and repository are not the same, make sure to
-> configure the `repository` correctly. If you don't, the repository
-> location will be inferred from the app name and the deploy will fail.
-> It must be set after the `load` statements to override the defaults.
->
->     # myapp/config/deploy.rb
->     load 'defaults'
->     load 'ruby'
->
->     set :repository, 'git@github.com/alphagov/foo.git'
->
-> If you don't load deploy/assets in your deploy script, none of your
-> applications assets (css, images) etc will be copied to your app's
-> public folder when it's deployed. If the asset isn't in public it
-> can't be rendered.
->
->     load 'deploy/assets'
+```
+desc "Run govuk-lint on all files"
+task "lint" do
+  sh "govuk-lint-ruby --format clang"
+  sh "govuk-lint-sass app/assets/stylesheets"
+end
+```
+
+Then add `task default: %i(spec lint)` in Rakefile and finally run `rake`.
+
+## Puppet, DNS, Sentry and Beyond
+
+  * To prepare the servers for your app, follow the doc in [govuk-puppet]
+  * To configure app deployments, follow the docs in [govuk-app-deployment]
+  * To enable external DNS, follow the article in the [Developer Docs](dns)
+  * Make a PR to add your app to `data/applications.yml` for these docs :-)
```

Comment:
Might be nice that this is a link to the file?

Reply:
Done :-)